plugins/repoquery.py

```diff
@@ -132,45 +89,41 @@ def parse_arguments(args):
         const='unsatisfied', action='store_const',
         help=_('limit the query to installed packages with unsatisfied dependencies'))
 
+    display = parser.add_argument_group(_('display arguments'))
+    outform = display.add_mutually_exclusive_group()
+    # options with default must go first
+    outform.add_argument('--qf', "--queryformat", dest='queryformat',
+                         default=QFORMAT_DEFAULT,
+                         help=_('format for displaying found packages'))
+    outform.add_argument('-i', "--info", dest='queryformat',
+                         action='store_const', const=QUERY_INFO,
+                         help=_('show detailed information about the package'))
+    outform.add_argument('-l', "--list", dest='queryformat',
+                         action='store_const', const=QUERY_FILES,
+                         help=_('show list of files in the package'))
+    outform.add_argument('-s', "--source", dest='queryformat',
+                         action='store_const', const=QUERY_SOURCERPM,
+                         help=_('show package source RPM name'))
+
     help_msgs = {
-        'conflicts': _('Display capabilities that the package conflicts with.'),
-        'enhances': _('Display capabilities that the package can enhance.'),
-        'obsoletes': _('Display capabilities that the package obsoletes.'),
-        'provides': _('Display capabilities provided by the package.'),
-        'recommends':  _('Display capabilities that the package recommends.'),
-        'requires':  _('Display capabilities that the package depends on.'),
-        'suggests':  _('Display capabilities that the package suggests.'),
-        'supplements':  _('Display capabilities that the package can supplement.')
+        'conflicts': _('display capabilities that the package conflicts with'),
+        'enhances': _('display capabilities that the package can enhance'),
+        'obsoletes': _('display capabilities that the package obsoletes'),
+        'provides': _('display capabilities provided by the package'),
+        'recommends': _('display capabilities that the package recommends'),
+        'requires': _('display capabilities that the package depends on'),
+        'suggests': _('display capabilities that the package suggests'),
+        'supplements': _('display capabilities that the package can supplement')
     }
     for arg in ('conflicts', 'enhances', 'obsoletes', 'provides', 'recommends',
                 'requires', 'suggests', 'supplements'):
         name = '--%s' % arg
-        const = '%%{%s}' % arg
-        outform.add_argument(name, dest='queryformat', action='store_const',
-                             const=const, help=help_msgs[arg])
+        display.add_argument(name, dest='capabilities', action='append_const',
```

Comment:
I think it should rather go to `insearch`. It affects (or at least we have agreed that it should) searching while the format is affected only when `--resolve` is used.